src/main/java/org/folio/processing/mapping/mapper/MappingContext.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.processing.mapping.mapper;
+
+import org.folio.processing.mapping.defaultmapper.processor.parameters.MappingParameters;
+
+public class MappingContext {
```

Comment:
@RuslanLavrov  I think we should add there mappingRules also. They should be provided because of logic in md-inventory.

Reply:
@VRohach, default mapping rules is not used by mapping mechanism which is called through MappingManager#map(), that's why we should not add them to the mapping context